setup.py

```diff
@@ -9,29 +9,32 @@
 
 RE_REQUIREMENT = re.compile(r'^\s*-r\s*(?P<filename>.*)$')
 
-PYPI_RST_FILTERS = (
-    # Replace code-blocks
-    (r'\.\.\s? code-block::\s*(\w|\+)+', '::'),
-    # Remove travis ci badge
-    (r'.*travis-ci\.org/.*', ''),
-    # Remove pypip.in badges
-    (r'.*pypip\.in/.*', ''),
-    (r'.*crate\.io/.*', ''),
-    (r'.*coveralls\.io/.*', ''),
+PYPI_CLEANDOC_FILTERS = (
+    # Remove badges lines
+    (r'\n.*travis-.*', ''),
+    (r'\n.*requires-.*', ''),
+    (r'\n.*david-dm.*', ''),
+    (r'\n.*gitter-.*', ''),
+    (r'\n.*coveralls-.*', ''),
+    # Transform links
+    (r'\[(.+)\]\[(.+)\]', '`\g<1> <\g<2>_>`_'),
```

Comment:
Oh I see, mmmh. I think it's worth it to stay as standard as we can and avoid that kind of tricks.